lib/packwerk/cli.rb

```diff
@@ -49,181 +45,23 @@ def run(args)
 
     sig { params(args: T::Array[String]).returns(T::Boolean) }
     def execute_command(args)
-      subcommand = args.shift
-      case subcommand
-      when "init"
-        init
-      when "check"
-        output_result(parse_run(args).check)
-      when "update-todo", "update"
-        output_result(parse_run(args).update_todo)
-      when "validate"
-        validate(args)
-      when "version"
-        @out.puts(Packwerk::VERSION)
-        true
-      when nil, "help"
-        usage
+      command = args.shift || "help"
+      command_class = Commands.class_for(command)
```

Comment:
You can avoid the need for `class_for` by just `const_get`-ting the command (converted to class name), and rescuing the possible `NameError`, like this:

```ruby
      class_name = command.sub(" ", "_").underscore.classify + "Command"
      Commands.const_get(class_name).new(
        args,
        configuration: @configuration,
        out: @out,
        err_out: @err_out,
        progress_formatter: @progress_formatter,
        offenses_formatter: @offenses_formatter,
      ).run
    rescue NameError
      @err_out.puts("'#{command}' is not a packwerk command. See `packwerk help`.",)

      false
    end
```